proto/mint/v1beta1/mint.proto

```diff
@@ -26,23 +27,23 @@ message DistributionProportions {
     (gogoproto.moretags) = "yaml:\"staking\"",
     (gogoproto.nullable) = false
   ];
-  // pool_incentives defines the proportion of the minted minted_denom that is
-  // to be allocated as pool incentives.
-  string pool_incentives = 2 [
+  // community_pool defines the proportion of the minted mint_denom that is
+  // to be allocated to the community pool: growth.
+  string community_pool_growth = 2 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-    (gogoproto.moretags) = "yaml:\"pool_incentives\"",
+    (gogoproto.moretags) = "yaml:\"community_pool\"",
     (gogoproto.nullable) = false
   ];
-  // participation_rewards defines the proportion of the minted minted_denom that is
-  // to be allocated to participation rewards address.
-  string participation_rewards = 3 [
+  // community_pool defines the proportion of the minted mint_denom that is
```

Comment:
should this comment say "community_pool_security_budget defines..."?

Reply:
done